tests/Sniffs/Operators/data/NegationOperatorSpacingSniffRequireSpaceErrors.fixed.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+$a = $a-$b;
+$a = $a  -  $b;
```

Comment:
This may be outside the scope of this PR, but I'd expect this to be fixed to `$a = $a - $b;` without the double spaces. Spacing should be consistent, which this isn't.

I'll admit this may be done to allow aligning of multiple computations in a single block, but then I disagree with the idea of aligning expressions, especially when they may be complex: think of multiple computations with a variable number of operators; how do they align?